### PR TITLE
VIH-7166 create vmr with correct tile prefix

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-7166

### Change description ###

Update tile prefix for room participants

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts [hunk 1]
```diff
@@ -302,6 +302,8 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
     }
 
     toggleMuteParticipant(participant: PanelModel) {
+        console.warn('attempting to toggle mute participant');
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts:305]: console warn? can we use logger?

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html [hunk 2]
```diff
@@ -1,136 +1,210 @@
...
+      [text]="
+        (handRaised
+          ? 'private-consultation-room-controls.hand-raise-toggle-true'
+          : 'private-consultation-room-controls.hand-raise-toggle-false'
+        ) | translate
+      "
+      colour="grey"
+      (click)="toggleHandRaised()"
+    >
+      <span class="far fa-hand-paper" [class]="{ yellow: handRaised }"></span>
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html:130]: what fixed this?

**Review by levibutler — COMMENTED** [review]
👍

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts (hunk 1 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts:305]: oh bugger forgot to remove

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html (hunk 2 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html:130]: nothing, it's still the original implementation

**Review by levibutler — APPROVED** [review]

**Pull request merged**